__tests__/app.test.js

```diff
@@ -46,15 +58,47 @@ describe("GET /api/topics", () => {
         expect(data.topics).toEqual(idealOutputGetTopics);
       });
   });
+});
 
-  describe("Error handling for invalid endpoint", () => {
-    test("status:404, responds with an error message when passed an non-existant endpoint", () => {
-      return request(app)
-        .get("/api/topics/iDontExist")
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("Invalid endpoint.");
-        });
-    });
+describe("GET /api/articles/:article_id", () => {
+  test("Should be able to return an object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body;
+        expect(typeof articles).toBe("object");
+      });
+  });
+  test("Should be able to return an object with appropriate keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        let article = body.article;
+        expect(article.article_id).toBe(1);
+        expect(article.title).toEqual(expect.any(String));
+        expect(article.topic).toEqual(expect.any(String));
+        expect(article.author).toEqual(expect.any(String));
+        expect(article.body).toEqual(expect.any(String));
+        expect(article.created_at).toEqual(expect.any(String));
+        expect(article.votes).toEqual(expect.any(Number));
+      });
```

Comment:
You can afford to be more particular with your testing here. You know what article should come back